### DISCUSSION TITLE
Prior ownership claims

### DISCUSSION THREAD

**Discussion by whatrocks** [farcasterxyz/protocol#14] [General]
I like the idea of this one, but I think this could use more detail (or a comment that details will be added later):

> A prior ownership claim can be made by a user who owns a name on at least two major social media platforms if the current owner does not own the name on any major social media platforms.

* What counts as a "major" social media platform? Should it just say Twitter and XX (github?)
* What about common names like `charlie` - if I get to FC early and claim my first name because I'm super excited and into the community, can the person who's had `charlie` on Twitter and Github for the last 10 years come take it away?

**Comment by varunsrin** [comment]
Good q, this is intentionally vague and meant to be a placeholder. 

@danromero is working on a light-weight policy with the dev community which will be added here when it's done.
